src/Configuration.php

```diff
@@ -2,20 +2,25 @@
 namespace Wheniwork\Feedback;
 
 use Auryn\Injector;
-use Negotiation\NegotiatorInterface;
-use Spark\Resolver\ResolverInterface;
+use Equip\Configuration\ConfigurationInterface;
+use Equip\Env;
 
-class Configuration
+class Configuration implements ConfigurationInterface
 {
-    public function apply(Injector $injector, array $env)
+    private $env;
+
+    public function __construct(Env $env) {
+        $this->env = $env;
+    }
+
+    public function apply(Injector $injector)
     {
         // --------------------
         // Initialize responders
         // --------------------
-        $injector->delegate("\Spark\Responder\FormattedResponder", function(NegotiatorInterface $negotiator, ResolverInterface $resolver) {
-            $responder = new \Spark\Responder\FormattedResponder($negotiator, $resolver);
-            $responder = $responder->withFormatters([
-                'Spark\Formatter\JsonFormatter' => 1.0
+        $injector->prepare("Equip\Responder\FormattedResponder", function($responder, $injector) {
+            $responder = $responder->withData([
+                'Equip\Formatter\JsonFormatter' => 1.0
```

Comment:
The default is [the same][1], so this can be removed?

[1]: https://github.com/equip/framework/blob/e4f13955181eab18f1b0e42b1eb93933a35ce3cc/src/Responder/FormattedResponder.php#L34